Contents/mods/!pz_debugTools/media/lua/client/LuaExplorerPanel/LuaExplorerTab.lua

```diff
@@ -28,7 +28,13 @@ function AUDLuaExplorerTab:initialise(category)
     self.fileList.anchorRight = true;
     self.fileList.anchorBottom = true;
     self.fileList:initialise();
-    self.fileList:setOnMouseDoubleClick(self, CustomLuaFileExplorerList.onMouseDoubleClickFile);
```

Comment:
Move `self.fileList:setOnMouseDoubleClick()` after a check for `self.fileCategory`
- FAV and ALL Tabs reload a single file per double click
- MOD tab reload an entire mod folder